app/src/pages/inside/stepPage/modals/editDefectModals/makeDecisionModal/makeDecisionModal.jsx

```diff
@@ -146,6 +148,14 @@ const MakeDecision = ({ data }) => {
     !modalHasChanges && !!issueAction.nextAction && dispatch(hideModalAction());
     issueAction.nextAction && issueAction.nextAction();
   };
+  const moveToOptionsStep = () => {
+    setFormStep('options');
+  };
+
+  const moveToConfigurationStep = () => {
+    setFormStep('configuration');
```

Comment:
It's better to put `configuration` in a separate constant. The same for `options`.